lib/error.js

```diff
@@ -31,7 +31,7 @@ var ERRNO = {
   REQUEST_BLOCKED: 125,
   ACCOUNT_RESET: 126,
   INVALID_UNBLOCK_CODE: 127,
-  MISSING_TOKEN: 128,
+  // MISSING_TOKEN: 128,
```

Comment:
This is a convention that we already follow in this file, if you look a bit further up at the lines for `ACCOUNT_LOCKED` and `ACCOUNT_NOT_LOCKED`.

Once retired, we don't want to re-use old error numbers for new reasons because it may make diagnosing problems in the logs tricky around the boundaries.